src/api/Flooq.Api/Services/IContactService.cs

```diff
@@ -0,0 +1,58 @@
+using Flooq.Api.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace Flooq.Api.Services;
+
+public interface IContactService
+{
+  /// <summary>
+  /// Queries all <see cref="Contact"/>s from the Flooq database.
+  /// </summary>
+  /// <returns>All <see cref="Contact"/>s from the Flooq database.</returns>
+  Task<ActionResult<IEnumerable<Contact>>> GetContacts();
+
+  /// <summary>
+  /// Finds a specific <see cref="Contact"/> in the Flooq database according to a unique identification.
+  /// </summary>
+  /// <returns>The identified <see cref="Contact"/>.</returns>
+  /// <param name="id">The value which is used for identifying the <see cref="Contact"/>.</param>
+  Task<ActionResult<Contact?>>GetContact(string id);
```

Comment:
Again: string email, and please update comment accordingly.

Reply:
Done.